README.md

```diff
@@ -14,43 +14,32 @@
 
 ## Installation
 
-`$ npm i ml-integral-transforms`
+`$ npm i integral-transforms`
```

Comment:
It is published as ml-integral-transforms (and should be)

https://github.com/mljs/integral-transforms/blob/main/package.json#L2